src/config/IAppConfig.ts

```diff
@@ -47,6 +47,7 @@ export interface IAppConfig {
     oncoprintOncoKbHotspotsDefault?:"disable"|"custom";
     // Select hide VUS by default
     oncoprintHideVUSDefault?:boolean;
+    sessionServiceIsEnabled:boolean;
```

Comment:
this is passed in https://github.com/cBioPortal/cbioportal/pull/4111 - tests are only skipped if this is `false` so they will continue to run when this is undefined in older versions of the backend